runmariadb

```diff
@@ -33,33 +40,41 @@ if [[ -f "$MARIADB_CERT_FILE" && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "
 fi
 
 if [ ! -d "${DATADIR}/mysql" ]; then
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections "${MARIADB_MAX_CONNECTIONS}"
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_heap_table_size 1M
-    crudini --set "$MARIADB_CONF_FILE" mysqld innodb_buffer_pool_size 5M
-    crudini --set "$MARIADB_CONF_FILE" mysqld innodb_log_buffer_size 512K
-    crudini --set "$MARIADB_CONF_FILE" mysqld user "${USER}"
+    cat > "$MARIADB_CONF_FILE" <<EOF
+[mysqld]
+max_connections = "${MARIADB_MAX_CONNECTIONS}"
+max_heap_table_size = 1M
+innodb_buffer_pool_size = 5M
+innodb_log_buffer_size = 512K
+user = "${USER}"
+log_warnings = 4
+EOF
 
     # Config MariaDB to enable TLS
     if [ -f "$MARIADB_CERT_FILE" ]; then
-        crudini --set "$MARIADB_CONF_FILE" mariadb-10.3 ssl on
-        crudini --set "$MARIADB_CONF_FILE" mariadb-10.3 ssl_cert "${MARIADB_CERT_FILE}"
-        crudini --set "$MARIADB_CONF_FILE" mariadb-10.3 ssl_key "${MARIADB_KEY_FILE}"
+        cat >> "$MARIADB_CONF_FILE" <<EOF
+ssl
```

Comment:
`ssl` is not a proper configuration value for server AFAIK, it is not found in https://mariadb.com/docs/server/security/data-in-transit-encryption/enterprise-server/enable-tls/ for example. It can be found in client config only.

Reply:
I've seen it in some samples, and we used to use it, but you're probably right. I'll test without it.